flink-end-to-end-tests/flink-streaming-kinesis-test/src/test/java/org/apache/flink/streaming/kinesis/test/model/Order.java

```diff
@@ -57,4 +57,9 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(code, quantity);
     }
+
+    @Override
+    public String toString() {
```

Comment:
I suppose this change was only needed for debugging purposes?

Reply:
`toString` is actually always needed for assertions ;)